Mathlib/Topology/MetricSpace/Lipschitz.lean

```diff
@@ -581,6 +590,141 @@ end Metric
 
 end LipschitzOnWith
 
+namespace LocallyLipschitz
+variable [PseudoEMetricSpace α] [PseudoEMetricSpace β] [PseudoEMetricSpace γ] {f : α → β}
+
+/-- A Lipschitz function is locally Lipschitz. -/
+protected lemma of_Lipschitz {K : ℝ≥0} (hf : LipschitzWith K f) : LocallyLipschitz f := by
+  intro x
+  use K, univ
+  rw [lipschitzOn_univ]
+  exact ⟨Filter.univ_mem, hf⟩
```

Comment:
This might be a good exercise in golfing; as nothing interesting is happening here, you can probably get this down to a single line in term mode.